src/components/Inventory/withdraw/flow/SelectPurposeForm.tsx

```diff
@@ -86,21 +102,31 @@ export default function SelectPurposeForm(props: SelectPurposeFormProps): JSX.El
   const [germinatingQuantityWithdrawn, setGerminatingQuantityWithdrawn] = useState<number>();
   const [zones, setZones] = useState<any[]>([]);
   const [zoneId, setZoneId] = useState<number>();
-  const snackbar = useSnackbar();
   const [noReadySeedlings, setNoReadySeedlings] = useState<boolean>(false);
-  const { isMobile } = useDeviceInfo();
-  const theme = useTheme();
-  const classes = useStyles();
   const [selectedSubzone, setSelectedSubzone] = useState<SubzoneInfo>();
   const [selectedZone, setSelectedZone] = useState<ZoneInfo>();
   const [speciesMap, setSpeciesMap] = useState<{ [key: string]: string }>({});
+  const [projectRecord, setProjectRecord] = useState<{ projectId?: number }>({});
+
   const tz = useLocationTimeZone().get(selectedNursery);
   const [timeZone, setTimeZone] = useState(tz.id);
-  const nurseryV2 = isEnabled('Nursery Updates');
-  const plantingSites = useAppSelector(selectPlantingSites);
 
   const numericFormatter = useMemo(() => numberFormatter(user?.locale), [numberFormatter, user?.locale]);
 
+  const availableProjects = useMemo(
+    () =>
+      projects?.filter((project: Project) =>
+        batches.find(
```

Comment:
`some`? (code is functionally correct as it is)

Reply:
You're right, `some` is semantically better here